pycoal/mining.py

```diff
@@ -21,9 +21,9 @@
 import time
 
 # classes identified as proxies for coal mining
-proxy_class_names = [u'Schwertmannite BZ93-1 s06av95a=b',
-                   u'Renyolds_TnlSldgWet SM93-15w s06av95a=a',
-                   u'Renyolds_Tnl_Sludge SM93-15 s06av95a=a']
+proxy_class_names = [u'Schwertmannite BZ93-1         BECKb AREF',
```

Comment:
Again, this cannot change... we need to maintain backwards compatibility and **add** functionality not take away.